src/Symfony/Component/Security/Core/Validator/Constraints/UserPasswordValidator.php

```diff
@@ -21,12 +22,17 @@
 
 class UserPasswordValidator extends ConstraintValidator
 {
-    private $securityContext;
+    private $tokenStorage;
     private $encoderFactory;
 
-    public function __construct(SecurityContextInterface $securityContext, EncoderFactoryInterface $encoderFactory)
+    /**
+     * @param SecurityContextInterface|TokenStorageInterface
+     *
+     * Passing a SecurityContextInterface as a first argument was deprecated in 2.7 and will be removed in 3.0
```

Comment:
actually you can remove all these warnings statements in the phpdoc because as I said below `SecurityContextInterface extends TokenStorageInterface`. so this warning is irrelevant.